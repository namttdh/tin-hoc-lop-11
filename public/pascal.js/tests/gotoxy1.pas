program Sample;

uses crt;

begin
  ClrScr();
  GotoXY(5,10);
  WriteLn('Hello positioned world');
end.
