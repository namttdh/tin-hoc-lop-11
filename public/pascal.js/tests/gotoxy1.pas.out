[2J[10;5HHello positioned world
